apps/aecore/src/aec_mining.erl

```diff
@@ -67,10 +63,10 @@ mine(Block, Attempts, InitialNonce, MaxNonce) ->
 
 %% Internal functions
 
--spec get_txs_to_mine(trees()) -> {ok, list(signed_tx()), non_neg_integer()} | {error, term()}.
-get_txs_to_mine(Trees) ->
+-spec get_txs_to_mine() -> {ok, list(signed_tx()), non_neg_integer()} | {error, term()}.
+get_txs_to_mine() ->
     {ok, Txs0} = aec_tx_pool:peek(aec_governance:max_txs_in_block() - 1),
-    case create_coinbase_tx(Trees) of
+    case create_coinbase_tx() of
```

Comment:
I see that other transactions in the block are not passed to creation of the signed coinbase tx, as the amount is not part of the coinbase tx. Ok.

Reply:
Amount is part of coinbase tx. We simply don't need state trees to create coinbase tx. Amount is taken from simplified governance atm.